UI/Scheduler/Russian.lproj/Localizable.strings

```diff
@@ -398,6 +398,10 @@
 "From" = "От";
 "To" = "Кому";
 
+/* dates */
+"DateFrom" = "С";
+"DateTo" = "До";
+
```

Comment:
Please add it to EN Language (base) then this will be translated into [Transifex](https://app.transifex.com/alinto/sogo)

Reply:
Hi, done!